### PR TITLE
document the atlas listener

### PR DESCRIPTION
**Pull request by meirish**

### CODE REVIEW

File: website/source/docs/config/index.html.md [hunk 1]
```diff
@@ -91,9 +91,11 @@ sudo setcap cap_ipc_lock=+ep $(readlink -f $(which vault))
 
 ## Listener Reference
 
-For the `listener` section, the only supported listener currently
-is "tcp". Regardless of future plans, this is the recommended listener,
-since it allows for HA mode.
+For the `listener` section, the only required listener is "tcp".
+Regardless of future plans, this is the recommended listener,
+as it allows for HA mode. If you wish to use the Vault
+Enterprise interface in HashiCorp Atlas, you may add an "atlas" listener block
```
Comment (KFishner) [inline website/source/docs/config/index.html.md]: can we link `"atlas" listener block` to the "connecting to vault enterprise..." section?

File: website/source/docs/config/index.html.md [hunk 2]
```diff
@@ -125,6 +127,34 @@ The supported options are:
...
+  * `infrastructure` (required) - Used to provide the Atlas infrastructure name and
+      the SCADA connection. The format of this is `username/environment`.
+
+  * `node_id` (required) - The identifier for an individual node—used in
+      the Vault Enterprise dashboard.
+
+  * `token` (required) - A token from Atlas used to authenticate SCADA session. Generate
+      one in the [Atlas](https://atlas.hashicorp.com/settings/tokens).
+
+Additionally, the global `cluster_name` will be used to identify your cluster
```
Comment (KFishner) [inline website/source/docs/config/index.html.md]: how is `cluster_name` set? 

also to make sure I understand the hierarchy it is:
An infrastructure is a group of clusters. A cluster is a group of nodes.

File: website/source/docs/config/index.html.md (hunk 1 above)
Comment (meirish) [inline website/source/docs/config/index.html.md]: I think so, need to figure out how...

File: website/source/docs/config/index.html.md (hunk 2 above)
Comment (meirish) [inline website/source/docs/config/index.html.md]: it's a global config option (outside of the `listener` block) - I'll double check that we have it in the docs and will link to it here

and yes, that's the hierarchy

**Comment by meirish** [comment]
@KFishner updated in https://github.com/hashicorp/vault/pull/1960/commits/4699101173422cf831da541e00372790ec1aae61 

@vishalnayak would you mind taking a look at the `cluster_name` bit? See if there's anything that should be added?

Thanks!

File: website/source/docs/config/index.html.md [hunk 3]
```diff
@@ -125,6 +131,34 @@ The supported options are:
...
+  * `infrastructure` (required) - Used to provide the Atlas infrastructure name and
+      the SCADA connection. The format of this is `username/environment`.
+
+  * `node_id` (required) - The identifier for an individual node—used in
+      the Vault Enterprise dashboard.
+
+  * `token` (required) - A token from Atlas used to authenticate SCADA session. Generate
+      one in the [Atlas](https://atlas.hashicorp.com/settings/tokens).
+
+Additionally, the global [`cluster_name`](#cluster_name) will be used to identify your cluster
```
Comment (vishalnayak) [inline website/source/docs/config/index.html.md]: I think we can skip the word "global" here.

Comment (vishalnayak) [inline website/source/docs/config/index.html.md]: We overwrite the `cluster_name` while we bring up a Vault instance. So, operators should make sure that the custom `cluster_name` they set are consistent across all the cluster members. Can we mention this point as well?

Comment (meirish) [inline website/source/docs/config/index.html.md]: @vishalnayak good points, both addressed here: https://github.com/hashicorp/vault/pull/1960/commits/61c88389ed524d84ab8955faede2954b6c02ba2c#diff-26ef8f4e1d98766632b9ac7db3d80911R155

**Pull request merged**